astropy/visualization/wcsaxes/helpers.py

```diff
@@ -24,6 +28,56 @@
 }
 
 
+def add_compass(
```

Comment:
Is it okay to keep this function with the underscore beforehand as an internal use indicator? if so, where should I put the tests for this function? If not, and I should just put the calculation inside add_compass, how should I write a test that ensures the NP angle is computed properly?  It doesn't feel like the figure test really does this